lib/diagnostic.js

```diff
@@ -15,9 +15,18 @@
 
 const fs = require('fs'); // jshint ignore: line
 
-const sumLines = (filename, callback) => {
-  /* your response here */
-  callback();
+let fileName = process.argv[2];
+
+let sum = 0;
+let i = 0;
+
+const sumLines = (fileName, callback) => {
+  fs.readFileSync(process.argv[2]).toString().split("\n");
```

Comment:
Notice that 2 things are being passed INTO our function. That's an indicator that we need them in our function.